b2/sync/folder.py

```diff
@@ -113,6 +114,20 @@ def ensure_non_empty(self):
         if not os.listdir(self.root):
             raise Exception('Directory %s is empty' % (self.root,))
 
+    def _is_file_readable(self, local_path, reporter):
+        """
+        Check if a file exists and is readable
```

Comment:
I'd remove the docstring completely, because the function is trivial to understand and one day someone will add a third `if` statement down below and forget to update the docstring.